types/react-select/src/Async.d.ts

```diff
@@ -5,43 +5,50 @@ import manageState from './stateManager';
 import { GroupedOptionsType, OptionsType, InputActionMeta, OptionTypeBase } from './types';
 
 export interface AsyncProps<OptionType extends OptionTypeBase> {
-  /* The default set of options to show before the user starts searching. When
-     set to `true`, the results for loadOptions('') will be autoloaded.
-     Default: false. */
-  defaultOptions?: GroupedOptionsType<OptionType> | OptionsType<OptionType> | boolean;
-  /* Function that returns a promise, which is the set of options to be used
-     once the promise resolves. */
-  loadOptions: (inputValue: string, callback: ((options: OptionsType<OptionType>) => void)) => Promise<any> | void;
-  /* If cacheOptions is truthy, then the loaded data will be cached. The cache
-     will remain until `cacheOptions` changes value.
-     Default: false. */
-  cacheOptions?: any;
+    /* The default set of options to show before the user starts searching. When
+       set to `true`, the results for loadOptions('') will be autoloaded.
+       Default: false. */
+    defaultOptions?: GroupedOptionsType<OptionType> | OptionsType<OptionType> | boolean;
+    /* Function that returns a promise, which is the set of options to be used
+       once the promise resolves. */
+    loadOptions: (
+        inputValue: string,
+        callback: (
+            options: OptionsType<OptionType> | GroupedOptionsType<OptionType>
+        ) => void
+    ) => Promise<
+        OptionsType<OptionType> | GroupedOptionsType<OptionType>
+    > | void;
+    /* If cacheOptions is truthy, then the loaded data will be cached. The cache
+       will remain until `cacheOptions` changes value.
+       Default: false. */
+    cacheOptions?: any;
 }
 
 export type Props<OptionType extends OptionTypeBase, IsMulti extends boolean> = SelectProps<OptionType, IsMulti> & AsyncProps<OptionType>;
 
 export const defaultProps: Props<any, boolean>;
 
 export interface State<OptionType extends OptionTypeBase> {
-  defaultOptions?: OptionsType<OptionType>;
-  inputValue: string;
-  isLoading: boolean;
-  loadedInputValue?: string;
-  loadedOptions: OptionsType<OptionType>;
-  passEmptyOptions: boolean;
+    defaultOptions?: OptionsType<OptionType>;
```

Comment:
It seems our prettier configs differ 😂 

Would you mind adding a prettier config to the root of this package's typedefs so we dont see format-related diffs in reviews going forward?

Reply:
I usually prefer 2 spaces, the 4 spaces is DT specific (see second bullet point [here](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes)).

Reply:
Strange 🤔 - perhaps the repo is so large that the root-level prettier config didnt properly pick up in my editor?